tests/foreman/cli/test_template.py

```diff
@@ -74,6 +80,60 @@ def test_update_template_1(self):
         self.assertEqual(len(result.stderr), 0)
         self.assertEqual(updated_name, result.stdout['name'])
 
+    def test_create_template_with_location(self):
+        """@Test: Check if Template with Location can be created
+
+        @Feature: Template - Create
+
+        @Assert: Template is created and new Location has been assigned
+
+        """
+        try:
+            new_loc = make_location()
+            new_template = make_template({
+                'name': gen_string('alpha', 10),
+                'location-ids': new_loc['id'],
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+
+        self.assertIn(new_loc['name'], new_template['locations'])
+
+    def test_create_template_locked(self):
+        """@Test: Check that locked Template cannot be created
+
+        @Feature: Template - Create
+
+        @Assert: It is not allowed to create locked Template
+
+        """
+        with self.assertRaises(CLIFactoryError) as ex:
+            make_template({
+                'name': gen_string('alpha', 10),
+                'locked': 'true',
+            })
+        self.assertIn("This template is locked. Please clone it",
+                      ex.exception.message)
```

Comment:
@omaciel mentioned a locale issue. This message will be raised if locale settings are en_US, but other locales may trigger other messages. Thoughts?